extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDD.scala

```diff
@@ -70,12 +70,12 @@ class KinesisBackedBlockRDDPartition(
  */
 private[kinesis]
 class KinesisBackedBlockRDD[T: ClassTag](
-    @transient sc: SparkContext,
+    sc: SparkContext,
     val regionName: String,
     val endpointUrl: String,
-    @transient blockIds: Array[BlockId],
+    blockIds: Array[BlockId],
```

Comment:
@skyluc @rxin, according to #8433 "if the class parameter are used in methods, then fields are created." I'm not sure how this applies in this case, where `blockIds` is also the name of a field in the superclass. In #8433, it looks like the fix was just to rename `blockIds` to `_blockIds` in order to avoid any ambiguity (see https://github.com/apache/spark/commit/c1bc4f439f54625c01a585691e5293cd9961eb0c#diff-b936e296c84502986961fb815d642458L64). Therefore, I'm going to make the same fix here for consistency's sake.